.github/workflows/trigger-caraml-doc-sync.yml

```diff
@@ -0,0 +1,22 @@
+# This workflow triggers sync of docs to caraml-dev/docs
+
+name: Trigger CaraML Docs Sync
+
+on:
+  push:
+    tags:
+      - 'v[0-9]+.[0-9]+.[0-9]+*'
+
+jobs:
+  trigger-sync:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Trigger Remote Doc Sync Workflow
+        uses: caraml-dev/docs/.github/actions/trigger-remote-docs-sync@main
+        with:
+          module: 'router'
```

Comment:
Currently, `turing` docs are under `experiment` module (https://github.com/caraml-dev/docs/tree/main/module/experiment), is the plan here to keep Turing and XP separate or under the same module? And if we want it under the same module, then this workflow likely would need to change right?

Reply:
Hmm, I spoke to Eric about this yesterday and I think we were planning to put `turing` docs under 'Router` and `xp` docs under 'Experiment', both modules having the same level in the tree hierarchy. I'm open to using our existing structure of `xp` being under `turing` too; what do you all think? @krithika369

Reply:
Just noticed this - as we discussed during the stand up, for the public docs, I think the current structure makes more sense (as we will not be using the internal code names) than nesting Experiments under Routers.